catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -570,6 +577,20 @@ private String getBindingTypeToActivate(List<ServiceBindingType> bindingTypes) {
         return properties;
     }
 
+    private String getConfigStringProperty(Configuration config, String property) {
+        if (config.getProperties() == null) {
```

Comment:
Should you go ahead and validate both parameters for null? So check `config` and `property`?